ledger/src/blockstore_meta.rs

```diff
@@ -51,9 +51,85 @@ impl Default for ConnectedFlags {
     }
 }
 
+/// Now deprecated format for completed data indexes.
+///
+/// Replaced by [`CompletedDataIndexesV2`].
+pub type CompletedDataIndexesV1 = BTreeSet<u32>;
+/// New format for completed data indexes.
+///
+/// `BTreeSet` is inefficient for de/serialization. We
+/// are migrating to a bitvec to improve that.
```

Comment:
```suggestion
/// Legacy completed data indexes type; de/serialization is inefficent for a BTreeSet
///
/// Replaced by [`CompletedDataIndexesV2`].
pub type CompletedDataIndexesV1 = BTreeSet<u32>;

/// A fixed size BitVec offers fast lookup and fast de/serialization
```
nit: Can we update the tense in these comments a bit. They make sense in the context of looking at this PR, but maybe not so much in 12 months long after the PR has landed. Ie, "we are migrating" is hopefully "we migrated" in 12 months

I threw a quick suggestion up, but feel free to tweak it